sg_jira/bridge.py

```diff
@@ -97,7 +96,7 @@ def __init__(
         self._jira_user = jira_user
         options = (
             {"token_auth": jira_secret}
-            if jira_user == "None"
+            if not jira_user
```

Comment:
Any chance `jira_user` could still be a string None with this code?
Should we do `jira_user in [None, "None"] to be safe or pointless?

Reply:
I thought having a `"None"` string can cause confusion for users. How can we validate which is the best?

Validate against `[None, "None"]` looks safer. Thanks.